docs/modules/ROOT/pages/relay.adoc

```diff
@@ -79,10 +79,25 @@ In the example above, we are also using a `DefenderRelayProvider` for making cal
 [[speed]]
 === Speed
 
-Instead of the usual `gasPrice`, the Relayer accepts a https://www.npmjs.com/package/defender-relay-client#speed[`speed`] parameter, which can be `safeLow`, `average`, `fast`, or `fastest`. These values are mapped to actual gas prices when the transaction is sent or resubmitted and vary depending on the state of the network.
+Instead of the usual `gasPrice`, the Relayer accepts a <<relay-api-reference.adoc#send-transaction,speed>> parameter, which can be `safeLow`, `average`, `fast`, or `fastest`. These values are mapped to actual gas prices when the transaction is sent or resubmitted and vary depending on the state of the network.
 
 NOTE: Today, mainnet gas prices for each speed are obtained from https://ethgasstation.info/[ethgasstation], falling back to https://etherchain.org/tools/gasPriceOracle[etherchain] if the former is not available. We are experimenting with other gas price oracles in the meantime. Testnet gas prices are obtained from a call to `eth_gasPrice` to the network.
 
+[[gas-price]]
+=== Gas Price
+
+Alternatively, you may specify a **fixed gas price** for a transaction, by setting the <<relay-api-reference.adoc#send-transaction,gasPrice>> parameter. Defender Relay will never change a fixed gas price and transaction either will be mined with a provided `gasPrice` or replaced by a NOOP tx if it is not mined before <<valid-until, validUntil>> time.
```

Comment:
Something sounds odd about the last sentence. How about...

"Transactions with fixed gas price are either mined with the specified `gasPrice` or replaced with a NOOP transaction if they couldn't be mined before <<valid-until, validUntil>> time."